vscode-dotnet-runtime-library/src/Acquisition/ReleasesResult.ts

```diff
@@ -0,0 +1,49 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+
+// ReleasesResult: Relevant data on current released runtime version.
+// Required json format:
+//    {
+//       "releases-index": [ --> Array of release channels
+//        {
+//           "channel-version": "X.X", --> Major.Minor version this channel represents
+//           "latest-runtime": "X.X.X", --> Most recently released full version of the runtime
+//           "latest-sdk": "X.X.X", --> Most recently released full version of the SDK
+//           ...
+//        },
+//        ...
+//    }
+export class ReleasesResult {

```

Comment:
What is this file doing here?

Reply:
Ah I think I included it during merge accidentally a long while back but it got merged into one file because it was needed in several places